### PR TITLE
[GUI ] Fixing inconsistent tab transitions 

### LINKED ISSUE #174: Low priority - wallet tabs inconsistent transition animations.
The overview and send tabs transition instantly when activated by user click. The receive, address, and settings tabs have short fades when clicked. These should be consistent across the tabs.

### PR DESCRIPTION
**Pull request by MatWaller**
Adding `showEvent` & `hideEvent` slots `sendcoindialog` & `overviewpage`.

Page/tab transition animations are now consistent.

closes #174

### CODE REVIEW

File: src/qt/veil/addresseswidget.cpp [hunk 1]
```diff
@@ -257,6 +257,7 @@ void AddressesWidget::initAddressesView(){
     // Complete me..
 }
 
+
```
Comment (blondfrogs) [inline src/qt/veil/addresseswidget.cpp]: Random extra line

File: src/qt/veil/addresseswidget.cpp [hunk 2]
```diff
@@ -267,6 +268,7 @@ void AddressesWidget::showEvent(QShowEvent *event){
     a->setEasingCurve(QEasingCurve::InBack);
     a->start(QPropertyAnimation::DeleteWhenStopped);
     reloadTab(isOnMyAddresses);
+
```
Comment (blondfrogs) [inline src/qt/veil/addresseswidget.cpp]: Random extra line

File: src/qt/veil/addresseswidget.cpp [hunk 3]
```diff
@@ -280,6 +282,7 @@ void AddressesWidget::hideEvent(QHideEvent *event){
     a->start(QPropertyAnimation::DeleteWhenStopped);
     connect(a,SIGNAL(finished()),this,SLOT(hideThisWidget()));
 
+
```
Comment (blondfrogs) [inline src/qt/veil/addresseswidget.cpp]: Random extra line

**Comment by presstab** [comment]
@MatWaller can you rebase this to master. Looks like there are a few merge conflicts preventing this from being mergable.

**Pull request closed without merging**